src/views/GameView.vue

```diff
@@ -1071,6 +1075,12 @@ export default {
 		max-width: 80%;
 		height: 100%;
 	}
+	#history {
+		& #history-logs {
+			height: 80%;
+			overflow: scroll;
```

Comment:
I see a blank vertical and horizontal scroll bar again on my end. Not sure if this is just me. changing this to `auto` seems to fix it.

Reply:
Set to overflow:auto. Can you confirm works on your machine?

Reply:
Works now!